src/flamegraph/mod.rs

```diff
@@ -142,6 +143,11 @@ pub struct Options<'a> {
     /// Defaults to None.
     pub subtitle: Option<String>,
 
+    /// # of samples to size the flame graph width to
+    ///
+    /// Defaults to None, which means it will be determined by the input being charted.
```

Comment:
I think it'd be useful to document here why you might want to use this option as well.